src/index.js

```diff
@@ -446,7 +446,8 @@ export default class HyperScreen extends React.Component {
     const formData = new FormData();
     let formHasData = false;
 
-    ['text-area', 'text-field', 'select-single', 'select-multiple']
+    // TODO: It would be more flexible to grab any element with a name and value.
+    ['text-area', 'text-field', 'select-single', 'select-multiple', 'picker-field']
```

Comment:
We should probably move these to a constant (maybe to the top of the file or function) and create an enum type for these

Reply:
I don't think it's really a type, just a single list of elements.